<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QMainWindow" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#0A0C0E  ;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:#0A0C0E  ;</string>
   </property>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color: #ffcc00; 
    color: #333333; 
    font-size: 12px;
</string>
    </property>
    <property name="placeholderText">
     <string>اختیاری</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>phone:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">          background-color: #BF360C;
            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_address">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color: #ffcc00; 
    color: #333333; 
    font-size: 12px;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">            color: white;
            border-radius: 10px;
            font-size: 26px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_invitationcode">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>invitationcode:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gosignin">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>email :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(255, 255, 255);
    font-size: 14px;

</string>
    </property>
    <property name="text">
     <string>Have an account? Sign in now.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPhone">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color: #ffcc00; 
    color: #333333; 
    font-size: 12px;
</string>
    </property>
    <property name="placeholderText">
     <string>اختیاری</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddress">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color: #ffcc00; 
    color: #333333; 
    font-size: 12px;
</string>
    </property>
    <property name="placeholderText">
     <string>اختیاری</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    font-size: 16px;
    font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>user name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color:#ffcc00; 
    color: #333333; 
    font-size: 12px;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="invitationcode">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
    padding: 0px 12px; 
    background-color: #ffcc00; 
    color: #333333; 
    font-size: 12px;
</string>
    </property>
    <property name="placeholderText">
     <string>اختیاری</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>471</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>E:/Downloads/Jowhareh_galleries_poster_b232edcc-d05d-4849-9fd0-e503d64710f0.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
